src/components/Menu/MenuItem.tsx

```diff
@@ -60,28 +76,11 @@ export function MenuItem({
         parentSetOpen(true);
       }
     }
-  }, []);
+  }, [asPathWithoutHash]);
```

Comment:
Assuming this is a string, right? If it's an object, I have concerns about performance if we don't memoize it.

Reply:
Yea `asPathWIthoutHash` should be a string since I'm only using `router.asPath`. 

https://nextjs.org/docs/pages/api-reference/functions/use-router#router-object - (look for "asPath" and it should say the type is a string)